src/stactools/nisar_sim/metadata.py

```diff
@@ -106,66 +107,45 @@ def parse_ann(self, ann_href: str) -> Dict[str, Any]:
 
 class Metadata:
     def __init__(self, href: str, id: str, h5_metadata: Any, ann_metadata: Any) -> None:
+
         self.href = href
         self.base_id = "_".join(id.split("_")[:-2])
         self.h5_metadata = h5_metadata
         self.ann_metadata = ann_metadata
-
-        def _get_geometries() -> Tuple[List[float], Dict[str, Any]]:
-            h5_polygon = self.h5_metadata.metadata["science"]["LSAR"][
-                "identification"
-            ].get("boundingPolygon")
-            if not h5_polygon:
-                raise H5MetadataException(
-                    f"Unable to locate boundingPolygon for {self.h5_metadata.href}"
-                )
-            polygon = loads(h5_polygon[()])
-            geometry = mapping(polygon)
-            bbox = list(polygon.bounds)
-
+        
+        def _get_geometries_from_ann_metadata() -> Tuple[List[float], Dict[str, Any]]:
+            # using the A file for geometries. A and B are approximately similar. 
+            
+            def _get_rounded_coord(k) -> float:
+                return np.round(float(self.ann_metadata.metadata_a[k]), 4)
```

Comment:
There is a python basic round(), is there a difference between the np import round?

Reply:
I think we should keep 3 decimal digits

Reply:
done in the other PR